Frameworks/CoreGraphics/CGContextCairo.mm

```diff
@@ -105,6 +105,11 @@
 // An OSS request has been submitted. The link for the request is:
 // https://osstool.microsoft.com/palamida/RequestDetails.htm?rid=40072&projectId=1
 void CGContextCairo::_cairoImageSurfaceBlur(cairo_surface_t* surface) {
+    // TODO #1635: It's unsafe to edit the bitmap not through D2D1RenderTarget, during a BeginDraw()
```

Comment:
These are a fairly significant hit to performance for the non-text CGContextCairo drawing functions, eating up >50% of time in CGContextFillRect, for instance. This may be unavoidable until the CGD2D merge, and is a tradeoff - text-drawing is still a much bigger bottleneck (CGContextFillRect, while slower, still does not take up much time). Please comment if you have any objections.